src/index.js

```diff
@@ -1,10 +1,42 @@
 const express = require('express');
-const path = require('path');
 
-const app = express()
+const app = express();
+const cors = require('cors');
+const session = require('express-session');
 const port = process.env.SERVER_PORT || 5000;
+const { Pool } = require('pg');
+
+const pgSession = require('connect-pg-simple')(session);
+
+app.use(cors());
+
+app.use(session({
+    secret: 'mit tsl teacher moments',
+    resave: true,
```

Comment:
Out of curiosity, why do we want `resave` to be true?

Reply:
Good catch! It would be true if the store doesn't save the active session. In our case though, out store doesn't need this to be true [https://www.npmjs.com/package/connect-pg-simple#installation](), so I'll change this to false